plugins/home/backstage-plugin-home-rss/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@roadiehq/backstage-plugin-home-rss",
+  "version": "0.0.1",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
```

Comment:
There was introduced a new field to the plugins package.json the `backstage.role` this is a frontend-plugin could you pls add it to this one ?
```
  "backstage": {
    "role": "frontend-plugin"
  },
```